ilib-scanner.js

```diff
@@ -36,13 +36,14 @@ var dateSet = [
 ];
 
 var localeCalMap = {
-    "am" : "EthiopicDate",
-    "ar" : "IslamicDate",
-    "cop" : "CopticDate",
-    "fa" : ["PersianDate", "PersianAlgoDate"],
-    "he": "HebrewDate",
-    "th" : "ThaiSolarDate",
-    "zh": "HanDate"
+    "ET" : "EthiopicDate",
+    "TR" : "IslamicDate",
+    "SA" : "IslamicDate",
```

Comment:
Add IslamicDate for all countries that are majority Islamic: MA, DZ, TN, LY, SD, JO, LB, SY, IQ, YE, AE, OM, QA, BH, KW, AF, PK, TM, KG, BD

Reply:
I updated list. Please re-check it